ui/src/pages/OBProxy/helper.ts

```diff
@@ -0,0 +1,17 @@
+import type { CommonKVPair } from '@/api/generated';
+
+/**
+ * 
+ * @description Determines whether two parameter lists are different
+ */
+export const isDifferentParams = (
+  newParams: CommonKVPair[],
+  oldParams: CommonKVPair[],
+) => {
+  if (newParams.length !== oldParams.length) return true;
```

Comment:
This helper function has bugs in it.

If old parameters are 
```
aKey: aValue
bKey: bValue
```
and new parameters are
```
aKey: aValue
aKey: aValue
```
The function judges the two lists of parameters are the same.

It would be a reasonable choice to construct two mappings and compare them.

Reply:
Fixed.